packages/aws-cdk-lib/scripts/verify-stripped-exp.js

```diff
@@ -0,0 +1,159 @@
+// +------------------------------------------------------------------------------------------------
```

Comment:
Why not as a typescript file?

Reply:
> tinny build scripts

Sure, but this ain't so tiny with a decent cyclomatic complexity. Having types would reduce the chance of defects when the file is updated.

I wouldn't mind ts-node or simply producing a js file from the ts as usual, whichever is easier/pragmatic.